amplfi/train/models/flow.py

```diff
@@ -121,7 +138,7 @@ def test_step(self, batch, batch_idx) -> bilby.result.Result:
         # plot corner and skymap
         skymap_filename = test_outdir / "mollview.png"
         corner_filename = test_outdir / "corner.png"
-        fits_filename = test_outdir / "amplfi.skymap.fits"
+        # fits_filename = test_outdir / "amplfi.skymap.fits"
```

Comment:
Should this just be deleted?

Reply:
Lets keep this b/c at some point I'll re add ths in